src/Core/Plugin.php

```diff
@@ -186,10 +186,14 @@ protected static function _loadConfig()
         if (Configure::check('plugins')) {
             return;
         }
-        try {
-            Configure::load('plugins');
-        } catch (\Exception $e) {
+
+        $vendorFile = ROOT . DS . 'vendor' . DS . 'cakephp-plugins.php';
```

Comment:
Then you would probably have to define sth here in cakephp/app:

    define('VENDOR', ROOT . DS . 'vendor' . DS);